src/mocker.js

```diff
@@ -34,12 +36,12 @@ async function main() {
 
 function saveToSql(vocabulary, cases, events) {
   let combinedFile = '';
+  let additionalCaseAttrs = 10
```

Comment:
Does this get used anywhere?

Reply:
No, this was something I missed to clean up. Thanks for spotting.